src/browser/deferred.ts

```diff
@@ -3,7 +3,7 @@ import { BrowserWindow as BrowserWindowOF } from '../shapes';
 import { toSafeInt } from '../common/safe_int';
 import { clipBounds } from './utils';
 
-export { handleMove };
+export default { handleMove };
```

Comment:
we can't remove this, can we?

Reply:
how so?

Reply:
Now, we do `import { handleMove } from './deferred';`. But we already `export function handleMove(` export this function. So if we remove this default export of a map, the imports won't change. It looks like an extra line to me, hope I'm not missing something

Reply:
It might be extra, left it in for paranoia. IMO it's never a bad thing to provide a default export of all named exports

Reply:
I'm 100% fine if you know it's good to have it there. I'm not okay if paranoia wrote this line of code :)